### ISSUE TITLE
App breaks when bundled (missing authFilter import)

### ISSUE THREAD

**Issue by AdamWillden** [SpoonX/aurelia-authentication#44]
My bundle is missing authFilter:

``` js
...
      "github:polymer/mutationobservers@0.4.2/MutationObserver.js",
      "github:spoonx/aurelia-api@2.0.7.js",
      "github:spoonx/aurelia-api@2.0.7/config.js",
      "github:spoonx/aurelia-api@2.0.7/endpoint.js",
      "github:spoonx/aurelia-api@2.0.7/index.js",
      "github:spoonx/aurelia-api@2.0.7/rest.js",
      "github:spoonx/aurelia-auth@0.13.6.js",
      "github:spoonx/aurelia-auth@0.13.6/app.fetch-httpClient.config.js",
      "github:spoonx/aurelia-auth@0.13.6/authService.js",
      "github:spoonx/aurelia-auth@0.13.6/authUtils.js",
      "github:spoonx/aurelia-auth@0.13.6/authentication.js",
      "github:spoonx/aurelia-auth@0.13.6/authorizeStep.js",
      "github:spoonx/aurelia-auth@0.13.6/baseConfig.js",
      "github:spoonx/aurelia-auth@0.13.6/index.js",
      "github:spoonx/aurelia-auth@0.13.6/oAuth1.js",
      "github:spoonx/aurelia-auth@0.13.6/oAuth2.js",
      "github:spoonx/aurelia-auth@0.13.6/popup.js",
      "github:spoonx/aurelia-auth@0.13.6/storage.js",
      "npm:aurelia-animator-css@1.0.0-beta.1.1.1.js",
...
```

authFilter is required here: https://github.com/SpoonX/aurelia-auth/blob/e62c9eb9d39d674cc218ea390fa87600b7346c9f/src/index.js#L16

But is not imported, thus not bundled and so results in a 404.

I assume you just need to add `import './authFilter';` to fix this.

**Comment by RWOverdijk** [comment]
~~@AdamWillden This is by design. Not everyone uses the authFilter. You'll have to import it yourself.~~

You're right, we _are_ using it. Good catch.

**Comment by AdamWillden** [comment]
~~ah okay. Therefore keep this open so we can add a note to the docs!~~

Okay then! lol

**Comment by AdamWillden** [comment]
This needs opening again. Was broken by #61

**Comment by RWOverdijk** [comment]
@AdamWillden Na, fixed it again in #75

**Comment by AdamWillden** [comment]
Ah okay good!
